independent-publisher-connectors/VIES by European Commission/script.csx

```diff
@@ -0,0 +1,65 @@
+﻿#r "Newtonsoft.Json"
+
+using System.Net;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System.Xml;
+using System.IO;
+
+public class Script : ScriptBase
+{
+    public override async Task<HttpResponseMessage> ExecuteAsync()
+    {
+        
+        return await this.HandleForwardOperation().ConfigureAwait(false);
+
+    }
+
+    private async Task<HttpResponseMessage> HandleForwardOperation()
+    {
+        HttpResponseMessage response;
+        var requestBody = await this.Context.Request.Content.ReadAsStringAsync().ConfigureAwait(false);
+        var data = JObject.Parse(requestBody);
+        var countryCode = ((string)data["countryCode"]).Trim();
+        var vatNumber = ((string)data["vatNumber"]).Trim().Replace(" ", string.Empty).Replace("-", string.Empty).Replace(".", string.Empty);
+        
+        var url = "http://ec.europa.eu/taxation_customs/vies/services/checkVatService";
+        var xml = @"<s:Envelope xmlns:s='http://schemas.xmlsoap.org/soap/envelope/'><s:Body><checkVat xmlns='urn:ec.europa.eu:taxud:vies:services:checkVat:types'><countryCode>"+countryCode+"</countryCode><vatNumber>"+vatNumber+"</vatNumber></checkVat></s:Body></s:Envelope>";
+
+        try
+        {
+            using (var client = new WebClient())
```

Comment:
@tposzytek I came to know this during preparing your connector for production deployment that - The creation of an HTTP client is not allowed in custom code in any form either using HTTPClient or WebClient. While we work to find a safe way to achieve some of the functionality, we ask you to explore alternate ways to achieve similar functionality. For example, for this method, you can create a simple connector for europa.eu and use the output from the connector in this field. Alternatively, you can either create a self-managed functions app to do the same. Thirdly, if you are the owner of the backend API you do similar work in your backend.